benchmarks/django-workload/graphite/docker-graphite-statsd/README.md

```diff
@@ -0,0 +1,235 @@
+
+## Start Using Graphite & Statsd
+
+### Send Some Stats
+
+Let's fake some stats with a random counter to prove things are working.
+
+```sh
+while true; do echo -n "example:$((RANDOM % 100))|c" | nc -w 1 -u 127.0.0.1 8125; done
+```
+
+### Visualize the Data
+
+Open Graphite in a browser.
+
+* http://localhost:81/dashboard
+* http://localhost:81/render?from=-10mins&until=now&target=stats.example
+
+## Secure the Django Admin
+
+Update the default Django admin user account. _The default is insecure._
+
+  * username: root
+  * password: root
+  * email: root.graphite@mailinator.com
+
+First login at: http://localhost:81/account/login
+Then update the root user's profile at: http://localhost:81/admin/auth/user/1/
+
+## Change the Configuration
+
+Read up on Graphite's [post-install tasks](https://graphite.readthedocs.org/en/latest/install.html#post-install-tasks).
+Focus on the [storage-schemas.conf](https://graphite.readthedocs.org/en/latest/config-carbon.html#storage-schemas-conf).
+
+1. Stop the container `docker stop graphite`.
+1. Find the configuration files on the host by inspecting the container `docker inspect graphite`.
+1. Update the desired config files.
+1. Restart the container `docker start graphite`.
+
+**Note**: If you change settings in `/opt/graphite/conf/storage-schemas.conf`
+be sure to delete the old whisper files under `/opt/graphite/storage/whisper/`.
+
+---
+
+**Important:** Ensure your Statsd flush interval is at least as long as the highest-resolution retention.
+For example, if `/opt/statsd/config.js` looks like this.
+
+```
+flushInterval: 10000
+```
+
+Ensure that `storage-schemas.conf` retentions are no finer grained than 10 seconds.
+
+```
+[all]
+pattern = .*
+retentions = 5s:12h # WRONG
+retentions = 10s:12h # OK
+retentions = 60s:12h # OK
+```
+
+## Statsd Admin Management Interface
+
+A management interface (default on port 8126) allows you to manage statsd & retrieve stats.
+
+```sh
+# show all current counters
+echo counters | nc localhost 8126
+```
+
+[More info & additional commands.](https://github.com/statsd/statsd/blob/master/docs/admin_interface.md)
+
+## Secure Grafana
+
+Update the default Grafana admin account. _The default is insecure._
+
+  * username: admin
+  * password: admin
+  * email: admin@localhost
+
+First login at: http://localhost
+Then update the admin user's profile at: http://localhost/admin/users/edit/1
+
+## Connect Grafana to Graphite
+
+Visit http://localhost/datasources/new
+Then configure the Graphite data source with the URL http://localhost:81
+
+## A Note on Volumes
+
+You may find it useful to mount explicit volumes so configs & data can be managed from a known location on the host.
+
+Simply specify the desired volumes when starting the container.
+
+```
+docker run -d\
+ --name graphite\
+ --restart=always\
+ -v /path/to/graphite/configs:/opt/graphite/conf\
+ -v /path/to/graphite/data:/opt/graphite/storage\
+ -v /path/to/statsd:/opt/statsd\
+ hopsoft/graphite-statsd
```

Comment:
Same as the above comment. I think we should change this line to the cloudsuite repository (e.g., cloudsuite/django:graphite-statsd or any other proper one).

Reply:
This image is not explicitly used while running the django image. 
I have retained the original readme from hopsoft just as a reference.

This readme need not be read/used while deploying django workload